NGitLab/Impl/Json/EnumConverter.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using NGitLab.Models;
+
+namespace NGitLab.Impl.Json
+{
+    internal sealed class EnumConverter : JsonConverterFactory
+    {
+        public override bool CanConvert(Type type)
+        {
+            // AccessLevel will be serialized as int, using System.Text.Json's default enum serializer
+            return type.IsEnum && type != typeof(AccessLevel);
```

Comment:
Can't you use `JsonStringEnumConverter`?

Reply:
AFAIK `JsonStringEnumConverter` does not handle the `[EnumMember]` attribute.